composer.json

```diff
@@ -38,9 +40,10 @@
     "phpseclib/phpseclib": "2.0.*",
     "vlucas/phpdotenv": ">=2.4.0 <=3.0",
     "ext-json": "*",
-    "ext-curl": "*"
+    "ext-curl": "*",
+    "mridang/pmd-annotations": "^0.0.2"
   },
-  "scripts-dev": {
```

Comment:
Wouldn't removing `-dev`make those scripts run with every `composer install/update`?

Reply:
It is not a valid directive.